tutorials/models_mapping/plot_1_tensor_models.py

```diff
@@ -0,0 +1,337 @@
+
+
+"""
+Tensor Meshes
```

Comment:
Singular Mesh

Reply:
I think unless it is specified, most people would hear tensor mesh and assume there is only one mesh.